frontend/college-toolbox/src/components/ScholarshipCard.jsx

```diff
@@ -122,11 +122,31 @@ const ScholarshipCard = ({
   const handleResumeDelete = () => {
     if (!isDeletingResume) {
       setIsDeletingResume(true);
-    } else {
-      // Perform delete action here or reset state if canceled
-      setResumeFileName('');
-      setIsDeletingResume(false);
     }
+    const formData = new FormData();
+    formData.append('uploadedFile.name', resumeFileName);
+    
+    // Perform delete action here or reset state if canceled
+    fetch('/ScholarshipApplication/delete-resume', {
```

Comment:
Update this fetch with the API_URL
Example found below:
https://github.com/uprm-inso4116-2023-2024-S1/semester-project-college-toolbox/blob/main/frontend/college-toolbox/src/services/authentication.ts